### PR TITLE
[SDK-1481] Add RBAC client

### PR DESCRIPTION
**Pull request by nidal-stytch**
Linear Ticket: [SDK-1481](https://linear.app/stytch/issue/SDK-1481/[ios]-[b2b]-add-rbac-client)

## Changes:

1. Create the `RBACPolicy` object (and the containing objects) and add it to the `BootstrapResponseData`.
2. Fix the url path for the b2b bootstrap call, it is the same as the b2c bootstrap call so I had to make an exception for this specific url in `StytchB2BClient+Routes.swift`. In the long run we should unify the bootstrap call so that there is no separate logic for b2b vs b2c.
3. Add RBAC client with sync and async methods for accessing the RBAC policy information.
4. Unrelated: In the stytch demo app I removed the plist file used to configure the app.
5. Unrelated: Removed a section of the PR template.

## Checklist:
- [x] I have verified that this change works in the relevant demo app, or N/A
- [x] I have added or updated any tests relevant to this change, or N/A
- [x] I have updated any relevant README files for this change, or N/A

### CODE REVIEW

File: Sources/StytchCore/StytchB2BClient/StytchB2BClient+Routes.swift [hunk 1]
```diff
@@ -15,8 +15,14 @@ extension StytchB2BClient {
         case oauthRoute(OAuthRoute)
 
         var path: Path {
-            let (base, next) = routeComponents
-            return "b2b".appendingPath(base).appendingPath(next.path)
+            switch self {
+            case let .bootstrap(route):
```
Comment (jhaven-stytch) [inline Sources/StytchCore/StytchB2BClient/StytchB2BClient+Routes.swift:19]: oof, so all B2B bootstrap requests were getting the b2b prepended and thus never completing with accurate bootstrap data 😬 Glad you caught and fixed it!

File: Tests/StytchCoreTests/B2BMagicLinksTestCase.swift [hunk 2]
```diff
@@ -195,6 +195,22 @@ extension MemberSession {
             expiresAt: refDate.advanced(by: 60 * 60 * 24),
             authenticationFactors: [],
             customClaims: nil,
+            roles: ["reader"],
```
Comment (jhaven-stytch) [inline Tests/StytchCoreTests/B2BMagicLinksTestCase.swift:198]: I feel like all these mocks should be moved out of the test case and into their own file. Thoughts?

**Review by jhaven-stytch — APPROVED** [review]
LGTM!

File: Tests/StytchCoreTests/B2BMagicLinksTestCase.swift (hunk 2 above)
Comment (nidal-stytch) [reply Tests/StytchCoreTests/B2BMagicLinksTestCase.swift:198]: Yeah this is definitely something to do.

File: Sources/StytchCore/StytchB2BClient/StytchB2BClient+Routes.swift (hunk 1 above)
Comment (nidal-stytch) [reply Sources/StytchCore/StytchB2BClient/StytchB2BClient+Routes.swift:19]: Yeah, initially I couldn't for the life of me figure out why the calls weren't working.

**Pull request merged**